test/fixtures/netweaver_simple_example/main.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+resource "random_id" "random_suffix" {
+  byte_length = 4
+}
+
+locals {
+  gcs_bucket_name        = "post-deployment-bucket-${random_id.random_suffix.hex}"
+  gcs_bucket_static_name = "${var.nw_deployment_bucket}"
+}
+
+#TODO: Add creation of a network that's similar to app2
```

Comment:
This comment should be removed and an issue should be opened against the repository so that we can effectively track the work.

Reply:
Removed this comment